content/blog/command-provider-ga/index.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Announcing the general availability of the Command provider"
+
+date: 2024-06-20T11:02:20+02:00
+
+draft: false
+
+meta_desc: >-
+    Version v1 of the Pulumi Command provider marks the provider’s general availability (GA).
+    The release expands support for copying assets between local and remote hosts.
+
+meta_image: meta.png
+
+authors:
+    - thomas-kappler
+
+# None of the tags in BLOGGING.md seem to match, but `providers` exists already
+tags:
+    - providers
+
+# The social copy used to promote this post on Twitter and Linkedin. These
+# properties do not actually create the post and have no effect on the
+# generated blog page. They are here strictly for reference.
+
+# Here are some examples of posts we have made in the past for inspiration:
+# https://www.linkedin.com/feed/update/urn:li:activity:7171191945841561601
+# https://www.linkedin.com/feed/update/urn:li:activity:7169021002394296320
+# https://www.linkedin.com/feed/update/urn:li:activity:7155606616455737345
+# https://twitter.com/PulumiCorp/status/1763265391042654623
+# https://twitter.com/PulumiCorp/status/1762900472489185492
+# https://twitter.com/PulumiCorp/status/1755637618631405655
+
+social:
+    twitter:
+    linkedin:
+---
+
+Today, we’re happy to announce version v1.0.0 of the Pulumi Command provider. This release marks the provider’s official transition from preview status to general availability (GA).
+Since its first preview release in late 2021, many users found the flexibility of the Command package valuable. In fact, it’s currently our tenth-most popular provider, so it's time to make things official!
+
+<!--more-->
+
+The Pulumi Command Provider enables you to execute commands and scripts either locally or remotely as part of the Pulumi resource model, enabling stateful command execution. It also has convenient support for copying assets via SSH. Being able to run arbitrary commands provides great flexibility.
+
+Some of the provider's popular uses include:
+
+- Running a command locally after creating a resource, to register it with an external service
+- Running a command locally before deleting a resource, to deregister it with an external service
+- Running a command on a remote host immediately after creating it
+- Copying a file to a remote host after creating it (potentially as a script to be executed afterwards)
+- As a simple alternative to some use cases for Dynamic Providers, by running appropriate commands in the different stages of the Pulumi life cycle.
+
+Here’s a simple example of running commands when another resource changes. In this case, a file is created when a pet name is created or changed, and the file is deleted when the pet name is removed. In a real-world scenario, the command could register/deregister the resource with an external API.
+
+{{< chooser language "typescript,go,python,csharp,yaml" >}}
```

Comment:
I think we could use a more "exciting" example.  A few things:
1. Could we make one of the examples at https://www.pulumi.com/registry/packages/command/ simple enough to include here?
2. I think using `RandomPet` may be distracting - it's not a "normal" infrastructure thing, and not really core to what we're trying to talk about here.
3. If none of the existing examples above work, would still be good to describe a "real" scenario here of some sort

Reply:
replaced with a working Lambda example